src/main/java/uk/gov/ida/matchingserviceadapter/mappers/MatchingDatasetToMatchingDatasetDtoMapper.java

```diff
@@ -114,4 +115,9 @@ public VerifyAddressDto apply(Address input) {
                         input.isVerified()))
                 .collect(Collectors.toList());
     }
+
+    public static Comparator<TransliterableMdsValue> getFirstNameComparator() {
+        return Comparator.comparing(TransliterableMdsValue::isVerified, Comparator.reverseOrder())
```

Comment:
We could name this better. It's the comparator we are using for firstName, but it doesn't describe what it does.

Reply:
Any suggestions? `getComparatorThatPrioritisesVerifiedStatusThenCurrentAttributes()` feels a bit unwieldy... 😅

Reply:
How about comparatorByVerifiedThenCurrent